### ISSUE TITLE
exp: train a model on combined contacts-v1.sequence_only + contacts-v1 datasets

### ISSUE THREAD

**Issue by timodonnell** [Open-Athena/MarinFold#87]
## Kind

models

## Question

Is there benefit to including unlabelled (i.e. no structure available) protein sequences during training?

## Hypothesis

After an epoch through the training set, we'll see better eval losses on structure prediction with the extra sequence-only data than without

## Background

Take a look at model training experiment #67 - we start with these parameters

We have a large corpus of sequence-only data in the contacts-v1 document structure here: https://huggingface.co/buckets/open-athena/MarinFold/tree/data/document_structures/contacts-v1.sequence_only

Our regular (i.e. with structure) contacts-v1 data is here: https://huggingface.co/buckets/open-athena/MarinFold/tree/data/document_structures/contacts_v1 

## Approach

Let's train for one epoch on the combined two datasets listed above. Make sure the datasets are sampled randomly (shuffled). Each batch should have a mix of the two types of data. Since we have more sequence-only data than with-sequence data, most of each batch will tend to be the sequence-only data.

We can plan to use v5p-8 like #67 but if bigger slices are available, it is fine to use those as well. If bigger slices are used scale the batch size proportionally to the increase in TPUs and multiply the learning rate by sqrt(batch size increase).

Make sure that eval losses for the two kinds of documents (i.e. sequence only and with sequence) are reported separately on wandb.

## Success criteria

We trained on the combined dataset for a single epoch without diverging
